registry/storage/linkedblobstore.go

```diff
@@ -95,10 +97,56 @@ func (lbs *linkedBlobStore) Put(ctx context.Context, mediaType string, p []byte)
 	return desc, lbs.linkBlob(ctx, desc)
 }
 
+type createOptions struct {
```

Comment:
At some point, we may want to unify these but this is acceptable for now.

@BrianBland Do you mind adding a godoc here?